feature/experimental/isis/ate_tests/lsp_updates_test/lsp_updates_test.go

```diff
@@ -50,7 +50,7 @@ func TestOverloadBit(t *testing.T) {
 	ts.PushAndStartATE(t)
 	ts.MustAdjacency(t)
 	setBit := isisPath.Global().LspBit().OverloadBit().SetBit()
-	deadline := time.Now().Add(time.Second)
+	deadline := time.Now().Add(time.Second * 10)
```

Comment:
why 10s?

Reply:
1 second is too short, test might fail due to deadline exceeded if timely reply is not received at controller.